test/client.js

```diff
@@ -931,7 +931,8 @@ describe('Client', () => {
             const a = { b: 1 };
             a.a = a;
 
-            const err = await expect(client.request({ method: 'POST', path: '/', payload: a })).to.reject('Converting circular structure to JSON');
+            const err = await expect(client.request({ method: 'POST', path: '/', payload: a })).to.reject();
+            expect(err.message).to.startWith('Converting circular structure to JSON');
```

Comment:
This is a fix for node 12, as seen https://travis-ci.org/hapijs/nes/jobs/537797710

This is unrelated to my PR.

![image](https://user-images.githubusercontent.com/295619/58424487-67b36700-808f-11e9-81fc-eee20d96324a.png)